### DISCUSSION TITLE
Dashboard List Panel: support multi-column display

### DISCUSSION THREAD

**Discussion by eraac** [grafana/grafana#38400] [Dashboards, Panels & Visualizations]
**What would you like to be added**: 
Dashboard List Panel should display the dashboard list in a multiple column way instead of having a long scroll

**Why is this needed**:
Just UX? Actually we have some "home" dashboards, that regroup all dashboards to a specific scope, and some of there have this panel with ~30 dashboards

**Comment by peterholmberg** [comment]
Hi @Eraac ! Could you provide a screenshot of what you mean by multi-column display?

**Comment by eraac** [comment]
@peterholmberg yes

Actually if the panel has weight of 2, 3, 4, ... 24 it's display only 1 dashboard per line
![image](https://user-images.githubusercontent.com/5497623/90725895-24ab6800-e2c1-11ea-9a45-69b3d4044af0.png)

That will be great if the list can be break into multi-column like this (don't judge my photoshop skill)
![image](https://user-images.githubusercontent.com/5497623/90726105-79e77980-e2c1-11ea-82aa-b5e533ae7b68.png)

It's an example with 2 columns, but more will be great (and configurable)

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
